chain-signatures/node/src/mesh/connection.rs

```diff
@@ -186,4 +197,20 @@ impl Pool {
             Err(_) => Err(FetchParticipantError::Timeout),
         }
     }
+
+    async fn test_empty_msg(
```

Comment:
let's rename this to `send_empty_msg` instead since we should reserve `test_*` functions for testing purposes